docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -295,58 +291,7 @@ curl -L -X GET 'http://localhost:8080/v2/topology' \
   </summary>
 </details>
 
-  </TabsItem>
-  <TabsItem value="zbctl" label="zbctl">
-
-1. Use the [zbctl client](/apis-tools/community-clients/cli-client/index.md) to retrieve list of remaining brokers
-
-```bash
-kubectl --context $CLUSTER_SURVIVING port-forward services/$HELM_RELEASE_NAME-zeebe-gateway 26500:26500 -n $CAMUNDA_NAMESPACE_SURVIVING
-zbctl status --insecure --address localhost:26500
-```
-
-<details>
-  <summary>Example output</summary>
-  <summary>
-
-```bash
-Cluster size: 8
-Partitions count: 8
-Replication factor: 4
-Gateway version: 8.6.0
-Brokers:
-  Broker 0 - camunda-zeebe-0.camunda-zeebe.camunda-london.svc:26501
-    Version: 8.6.0
-    Partition 1 : Leader, Healthy
-    Partition 6 : Follower, Healthy
-    Partition 7 : Follower, Healthy
-    Partition 8 : Follower, Healthy
-  Broker 2 - camunda-zeebe-1.camunda-zeebe.camunda-london.svc:26501
-    Version: 8.6.0
-    Partition 1 : Follower, Healthy
-    Partition 2 : Follower, Healthy
-    Partition 3 : Follower, Healthy
-    Partition 8 : Leader, Healthy
-  Broker 4 - camunda-zeebe-2.camunda-zeebe.camunda-london.svc:26501
-    Version: 8.6.0
-    Partition 2 : Follower, Healthy
-    Partition 3 : Leader, Healthy
-    Partition 4 : Follower, Healthy
-    Partition 5 : Follower, Healthy
-  Broker 6 - camunda-zeebe-3.camunda-zeebe.camunda-london.svc:26501
-    Version: 8.6.0
-    Partition 4 : Follower, Healthy
-    Partition 5 : Follower, Healthy
-    Partition 6 : Follower, Healthy
-    Partition 7 : Leader, Healthy
-```
-
-  </summary>
-</details>
-  </TabsItem>
-</Tabs>
-
-2. Port-forward the service of the Zeebe Gateway to access the [management REST API](../../zeebe-deployment/configuration/gateway.md#managementserver)
+1. Port-forward the service of the Zeebe Gateway to access the [management REST API](../../zeebe-deployment/configuration/gateway.md#managementserver)
```

Comment:
We should keep this page intact at this point. This guide is done in the "batteries included" manner, meaning the exact commands a user runs are prepared for them. Replacing them with a generic link to the API guide is not sufficient and also omits important info (like `kubectl` commands for port-forwarding etc.)

Reply:
@Langleu considering your work in https://github.com/camunda/c8-multi-region/pull/158/files, let me know if you think differently.

Reply:
The `batteries included` raised by Maxim is not in danger by removing zbctl as we provide the REST alternatives since 8.6 and had the zbctl stuff in tabs for easy removal in the future.

Just re-read the [announcement](https://camunda.com/blog/2024/09/deprecating-zbctl-and-go-clients/), so it seems like it would make sense to remove it from 8.7 docs.

The page does still have some `zbctl` mentionings, which would make sense to remove then altogether.